label_studio/organizations/serializers.py

```diff
@@ -104,6 +98,23 @@ class Meta:
         fields = ['id', 'organization', 'user']
 
 
+class OrganizationMemberSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
+    annotations_count = serializers.SerializerMethodField(read_only=True)
+    contributed_projects_count = serializers.SerializerMethodField(read_only=True)
+
+    def get_annotations_count(self, member):
+        org = self.context.get('organization')
+        return member.user.annotations.filter(project__organization=org).count()
```

Comment:
I am a little worried about the performance of this count. This will execute a new query for every user in the organization.
Is this similar to how the django template was handling this? If that is the case, then we can address performance optimization in a future ticket.

Reply:
Oh I just realized this is for the "detail" view so this would be the count for only one org member. This is totally fine!

Reply:
In the [template](https://github.com/HumanSignal/label-studio/blob/7479e3e82cc78e97d42949f414813629ddde84b3/label_studio/users/templates/users/user_account.html#L214-L215) is using these two methods in the user model. The only difference is that in the model is for the active organization, here is for the provided one.